### PR TITLE
hero section

### PR DESCRIPTION
**Pull request by muhammedshaheer**

### CODE REVIEW

File: src/button-group/ButtonGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { ReactNode } from 'react';
+
+import className from 'classnames';
+
+type IButtonGroupProps = {
+  xl?: boolean;
```
Comment (jazirk) [inline src/button-group/ButtonGroup.tsx:6]: xl is a size?

File: src/navigation/NavbarThreeColumns.tsx [hunk 2]
```diff
@@ -3,24 +3,29 @@ import { ReactNode } from 'react';
...
 type INavbarProps = {
-  logo: ReactNode;
+  logo: string | undefined;
   children: ReactNode;
+  button?: ReactNode;
 };
 
-const NavbarTwoColumns = (props: INavbarProps) => (
-  <div className="flex flex-wrap justify-between items-center">
+const NavbarThreeColumns = (props: INavbarProps) => (
```
Comment (jazirk) [inline src/navigation/NavbarThreeColumns.tsx:11]: Just NavBar ?

**Review by jazirk — APPROVED** [review]
LGTM

File: src/button-group/ButtonGroup.tsx (hunk 1 above)
Comment (muhammedshaheer) [reply src/button-group/ButtonGroup.tsx:6]: yes like a different variant

File: src/navigation/NavbarThreeColumns.tsx (hunk 2 above)
Comment (muhammedshaheer) [reply src/navigation/NavbarThreeColumns.tsx]: updated

**Pull request merged**